package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "material-tailwind-kit",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@craco/craco": "^6.1.1",
+    "@material-tailwind/react": "0.2.0",
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "^5.2.0",
+    "react-scripts": "4.0.3",
+    "web-vitals": "^1.0.1"
+  },
+  "scripts": {
+    "start": "craco start",
+    "build": "craco build",
+    "test": "craco test",
+    "eject": "react-scripts eject"
```

Comment:
Hey dude. I usually like to add one more script here like:
```
"install:clean": "rm -rf node_modules/ && rm -rf package-lock.json && npm install && npm start"
```
You could add two of them:
```
"npm:clean": "rm -rf node_modules/ && rm -rf package-lock.json && npm install && npm start",
"yarn:clean": "rm -rf node_modules/ && rm -rf package-lock.json && yarn install && yarn start"
```
For both, `yarn` and `npm`. Just a thought. Add them if you think they might be useful.